docs/explanations/what_is_stardust/what_is_stardust.md

```diff
@@ -12,13 +12,63 @@ image: /img/logo/preview.png
 
 # What is Stardust?
 
-Stardust is a comprehensive utility upgrade of the IOTA Chrysalis protocol. The main goals of Stardust upgrade are to:
- - make IOTA an **infrastructure and settlement layer** for [second-layer smart contract chains](https://wiki.iota.org/smart-contracts/overview),
- - transform IOTA into a **multi-asset ledger** with custom, user defined tokens.
+The previous IOTA protocol, Chrysalis, was optimized for a single application: sending digital money from A to B.
+The upcoming Stardust upgrade introduces computation and utility to the ledger:
+- Making IOTA an **infrastructure and settlement layer** for [second-layer smart contract chains](https://wiki.iota.org/smart-contracts/overview) using [IOTA Smart Contracts (ISC)](#isc).
+- Transforming IOTA into a **multi-asset ledger** with custom, user-defined tokens using the new [tokenization framework](#tokenization-framework).
 
-The previous IOTA protocol, Chrysalis, is optimized for a single application: sending digital money from A to B.
-Stardust makes it possible to program that digital money with any business logic via smart contract chains, furthermore,
-you can also print your own unique digital money!
+## Tokenization Framework
+The tokenization framework enables many new functionalities that builders want, most importantly, the ability to mint other tokens and NFTs. Our approach is different from the approach used in most other platforms. 
```

Comment:
```suggestion
## Tokenization Framework

The tokenization framework enables many new functionalities that builders want, most importantly, the ability to mint other tokens and NFTs. Our approach is different from the approach used in most other platforms.
```

I would suggest to describe our approach or explain the difference briefly before jumping to advantages and disadvantages. For instance:

"The tokenization framework extends the protocol functionality just enough to mind native tokens and NFTs directly on Level 1, unlike other protocols that require a full smart contract to do so. It has its advantages and disadvantages."

This way the reader would know what exactly do we compare and could see where these advantages come from.

Reply:
Makes sense. But the full smart contracts sentence may be too much, as with the tokenization framework we planned to enable Smart contracts in the first place. We now just also make more use of our own L1 by adding native assets